src/components/ImagePicker/__snapshots__/ImagePicker.test.js.snap

```diff
@@ -165,7 +166,7 @@ exports[`Image add form ImagePicker component should render correctly with data
       "hel-event-languages": "Tapahtuman kielet",
       "hel-target-groups": "Kohderyhmät",
       "image-caption-limit": "Kuvaus eli kuvan alt-teksti (korkeintaan {limit} merkkiä)",
```

Comment:
This should be changed. Alt-text is now a separate field.

Reply:
I removed the translation as it wasn't used anywhere